drivers/flash/Kconfig.sc

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2025 Space Cubics, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig SC_QSPI_NOR
+	bool "Space Cubics QSPI driver"
+	default y
+	depends on DT_HAS_SC_QSPI_NOR_ENABLED
+	select FLASH_HAS_DRIVER_ENABLED
+	select FLASH_HAS_EXPLICIT_ERASE
+	select FLASH_HAS_PAGE_LAYOUT
+	help
+	  Enable support for Space Cubics NOR QSPI driver.
+
+if SC_QSPI_NOR
+
+config SC_QSPI_NOR_IDLE_TIMEOUT_MS
+	int "Timeout duration for waiting until the QSPI Bus becomes idle.[ms]"
+	default 100
+	help
+	  Timeout duration for waiting until the QSPI Bus becomes idle. [ms]
```

Comment:
Is this hardware property or a value user want to change?

is there valid range for this value? IOW, can we set any value for this?